src/app/extensions/quoting/shared/quote-interactions/quote-interactions.component.html

```diff
@@ -6,8 +6,13 @@
         data-testing-id="submit-quote-request"
         type="submit"
         class="btn btn-primary float-right"
-        [disabled]="quote.items.length === 0"
-        (click)="submit()"
+        [ngClass]="{ disabled: quote.items.length === 0 }"
+        [attr.aria-disabled]="quote.items.length === 0"
```

Comment:
pls refactor - all buttons and input.btn html elements which are aria-disabled should be styled like a disabled button, this should be done in the buttons.scss. the ngClass is not needed or only needed in exceptional cases